itest/burn_test.go

```diff
@@ -157,9 +157,10 @@ func testBurnAssets(t *harnessTest) {
 
 	// We'll now assert that the burned asset has the correct state.
 	burnedAsset := burnResp.BurnProof.Asset
-	allAssets, err := t.tapd.ListAssets(
-		ctxt, &taprpc.ListAssetRequest{IncludeSpent: true},
-	)
+	allAssets, err := t.tapd.ListAssets(ctxt, &taprpc.ListAssetRequest{
+		IncludeSpent:  true,
+		ScriptKeyType: allScriptKeysQuery,
```

Comment:
Do we need to change CLI defaults as a result of this? 

Also something to call out in the release notes as a change in behavior here.

Reply:
No, the defaults remain the same. But I added a new `--script_key_type` flag to all relevant commands so the users can also filter on the command line.